### ISSUE TITLE
Failed to start up ontology with '--testmode'         

### ISSUE THREAD

**Issue by zhongqiuwood** [ontio/ontology#291]
```
$ ./ontology --testmode             
2018/06/10 15:05:09.612818 [INFO ] GID 1, Config init success
Password:
2018/06/10 15:05:11.136472 [INFO ] GID 1, Wallet init success
2018/06/10 15:05:11.152077 [INFO ] GID 1, InitCurrentBlock currentBlockHash d3e758ab7b71216595d46dfa2771290b8ba172588639c47ef1b549eda882f846 currentBlockHeight 0
2018/06/10 15:05:11.152425 [ERROR] GID 1, Init ledger error:InitLedgerStoreWithGenesisBlock error GenesisBlock arenot init correctly
```

**Comment by Arbio5zt** [comment]
Before chain restart, there need to clear the old chain data first, i.e. manually delete the Chain lib,
ps: We have planed to add automatic clear feature in the later version.

**Comment by Arbio5zt** [comment]
will improved by #306 .

**Comment by voodoo12345** [comment]
fixed by #306
